requirements.txt

```diff
@@ -1,4 +1,5 @@
-django-councilmatic[convert_docs]>=3.0
+# django-councilmatic[convert_docs]>=3.0
+git+https://github.com/datamade/django-councilmatic.git@hcg/configurable-headshot-backend#egg=django-councilmatic
```

Comment:
TODO: Update this to new `django-councilmatic` release once it becomes available.